migrations/iql6bkdk.skill-category-is-activated.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE skill ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT false;
+ALTER TABLE skill_category ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT false;
```

Comment:
I think it would make more sense for the default here to be true, since it's unlikely someone would add a category and immediately inactivate it.